src/components/Chips/index.tsx

```diff
@@ -1,89 +1,107 @@
-    "font-light",  
+    "font-light",
     "items-center",
   ],
-    {
-      variants: {
-        variant: {
-          default: "",
-          not_active: "",
-          active: "",
-          addRightIcon : false,
-    addLeftIcon : false,
-        },
-        size: {
-          sm: "px-4 py-2 text-sm",
-          md: "px-4 py-2 text-base",
-          lg: "px-6 py-3 text-lg",
-        },
-        colorscheme: {
-          primary: "text-white",
-        },
+  {
+    variants: {
+      variant: {
+        [ChipsTypes.default]: "",
+        [ChipsTypes.not_active]: "",
+        [ChipsTypes.active]: "",
+        addRightIcon: false,
+        addLeftIcon: false,
+      },
+      size: {
+        [ChipsSize.small]: "px-3 py-0.5   text-sm",
+        [ChipsSize.medium]: "px-4 py-2 text-base",
+        [ChipsSize.large]: "px-6 py-3 text-lg",
       },
-      compoundVariants: [
-        {
-          variant: "default",
-          colorscheme: "primary",
-          className: "bg-[#D0E3FF] hover:bg-primary-600 text-[#334EAC] gap-2 ",
-        },
-        {
-          variant: "not_active",
-          colorscheme: "primary",
-          className:
-            "text-[#081F5C] bg-[#E8EBED] gap-2",
-        },
-        {
-          variant: "active",
-          colorscheme: "primary",
-          className: "text-primary-600 bg-gradient-to-r from-[#7096D1] to-[#334EAC] text-[#FFFFFF] gap-2",
-        },
-      ],
-      defaultVariants: {
-        variant: "default",
-        size: "sm",
+      colorscheme: {
+        primary: "text-white",
+      },
+    },
+    compoundVariants: [
+      {
+        variant: ChipsTypes.default,
         colorscheme: "primary",
+        className: "bg-[#D0E3FF]  text-[#334EAC] gap-2 ",
       },
-    }
-  );
+      {
+        variant: ChipsTypes.not_active,
+        colorscheme: "primary",
+        className:
+          "text-[#081F5C] bg-[#E8EBED] gap-2",
+      },
+      {
+        variant: ChipsTypes.active,
+        colorscheme: "primary",
+        className: "text-primary-600 bg-gradient-to-r from-[#7096D1] to-[#334EAC] text-[#FFFFFF] gap-2",
+      },
+    ],
+    defaultVariants: {
+      variant: ChipsTypes.default,
+      size: ChipsSize.small,
+      colorscheme: "primary",
+    },
+  }
+);
 
-  
 
 
-type ChipsProps = ComponentProps<"div"> & VariantProps<typeof chipsStyle>&{ addRightIcon?: boolean;
-    addLeftIcon?: boolean;}
+
+type ChipsProps = ComponentProps<"div"> & VariantProps<typeof chipsStyle> & {
+  addRightIcon?: boolean;
+  addLeftIcon?: boolean; 
+  variant?: ChipsTypes;
+  children?: ReactNode;
+};
 
 export const Chips = forwardRef<HTMLDivElement, ChipsProps>(
-  ({ variant = "default", size, colorscheme, className, addRightIcon = false, addLeftIcon = false,children, ...props }, ref) => {
+  ({ variant = ChipsTypes.default, size, colorscheme, className, addRightIcon = false, addLeftIcon = false, children, ...props }, ref) => {
+    let hasText = false;
+
+    switch (true) {
```

Comment:
what is this ??

Reply:
on which line that switch case ya variant type ?